### PR TITLE
(maint) Document 0.13 Release

### PR DESCRIPTION
**Pull request by jpogran**
Should document:

- Outline view
- Breadcrumbs
- Go to symbol
- Loading indicator
- More...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,8 +51,12 @@ A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marke
 
 This extension provides full Puppet Language support for [Visual Studio Code](https://code.visualstudio.com/).
 
+- Loading indicator
```
Comment (michaeltlombardi) [inline README.md]: Does this belong here? Seems reasonable to leave it out of the readme (but include in the changelog!) - not really a Puppet-centric feature, I think?

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,13 +10,15 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 - ([GH-366](https://github.com/lingua-pupuli/puppet-vscode/issues/366)) New Debug Adapter
 - ([GH-354](https://github.com/lingua-pupuli/puppet-vscode/issues/354)) Document PDK as source information
+- ([GH-385](https://github.com/lingua-pupuli/puppet-vscode/issues/385)) New OutlineView, Breadcrumb, and Go to symbol Features
```
Comment (michaeltlombardi) [inline CHANGELOG.md]: Not: Symbol should be capitalized for consistency.

File: README.md [hunk 3]
```diff
@@ -85,7 +92,14 @@ You can find instructions and installation links here:
 - MacOSX
 - Linux
 
-## Usage 
+## Usage
+
+### Loading indicator
+
+The Puppet extension now provides more information while loading, so you now what features are enabled as they are finished loading
```
Comment (michaeltlombardi) [inline README.md]: Suggest rewording this to something like:

> The Puppet extension includes additional information in the form of a tooltip which describes the features that are loaded and still loading during startup.

File: README.md [hunk 4]
```diff
@@ -109,6 +123,24 @@ Hovering over any resource, class declaration or other Puppet symbol provides in
 
 ![hover_support](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/hover_support.gif)
 
+### Outline View
+
+The new `Outline View` shows the symbol tree of the currently selected Puppet manifest. This currently supports classes, types, variables and parameters. More symbols are planned to be added.
```
Comment (michaeltlombardi) [inline README.md]: Pseudo nit: One sentence per line (semantic linefeeds). This makes review (and editing different) much easier.

The last sentence should be reworked - maybe something like a check box list with the possible symbols and only those which are implemented are checked? Or a link to the issue? Or drop the sentence entirely and just update the section when new symbols are added?,

File: README.md [hunk 5]
```diff
@@ -109,6 +123,24 @@ Hovering over any resource, class declaration or other Puppet symbol provides in
...
+
+### Breadcrumbs
+
+When the `breadcrumbs.enabled` setting is set to true, both the file path and the symbol tree hierarchy are shown in the Breadcrumb view.
+
+![breadcrumbs](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/breadcrumbs.gif)
+
+### Go to Symbol
+
+Opening the `Command palatte` and typing the `@` symbol initiates the `Go to symbol` view which allows you to navigate around inside a Puppet manifest quickly.
```
Comment (michaeltlombardi) [inline README.md]: Palette and symbol should both be capitalized for consistency.

Suggest "... quickly" to "... more quickly"

File: README.md (hunk 1 above)
Comment (jpogran) [reply README.md]: I dunno. I think it's something nice to callout as it's helpful, but didn't know where to put it

File: README.md (hunk 4 above)
Comment (jpogran) [reply README.md]: While I generally agree on linefeeds, the whole document does not follow that proscription. I'm disinclined to acquiesce to your request here, and instead volunteer you to copy-edit the readme :)

Will work on last sentence

File: README.md (hunk 1 above)
Comment (michaeltlombardi) [reply README.md]: I think it can probably be excluded from here - the usage section below seems to handle it fine.

Comment (michaeltlombardi) [reply README.md]:
```suggestion
```

File: README.md (hunk 4 above)
Comment (michaeltlombardi) [reply README.md]:
```suggestion
The new `Outline View` shows the symbol tree of the currently selected Puppet manifest.
This currently supports classes, types, variables and parameters.
```
